### ISSUE TITLE
Allow changing the requested domain name for Telegram notifications, or allow it to use proxy

### ISSUE THREAD

**Issue by Lemonawa** [louislam/uptime-kuma#4654]
### 📑 I have found these related issues/pull requests

N/A :D

### 🏷️ Feature Request Type

Change to existing notification-provider

### 🔖 Feature description

Due to the block of GFW, we are unable to access api.telegram.org.
However, we can use reverse proxy to access it, like tgapi.lma.moe.

### ✔️ Solution

like https://github.com/whyour/qinglong 
![图片](https://github.com/louislam/uptime-kuma/assets/69106050/79ddbc63-1d31-4b19-90bb-bd73e7354e64)

**Comment by CommanderStorm** [comment]
Closing as a duplicate

Allowing for proxying of notifications is tracked in
- https://github.com/louislam/uptime-kuma/issues/616
- https://github.com/louislam/uptime-kuma/issues/2588

TL;DR: We'd love for such a [functionality to be contributed](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md), but [given the other things currently going on](https://github.com/louislam/uptime-kuma/milestones) like https://github.com/louislam/uptime-kuma/issues/4500 don't quite have the engineering resources to do so
=> If you want to get there faster, help would be appreciated ^^
